opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -2061,3 +2062,41 @@ def test_tracer_provider_init_default(self, resource_patch, sample_patch):
         sample_patch.assert_called_once()
         self.assertIsNotNone(tracer_provider._span_limits)
         self.assertIsNotNone(tracer_provider._atexit_handler)
+
+
+class TestRandomIdGenerator(unittest.TestCase):
+    _TRACE_ID_MAX_VALUE = 2**128 - 1
+    _SPAN_ID_MAX_VALUE = 2**64 - 1
+
+    @patch(
+        "random.getrandbits",
+        side_effect=[span.INVALID_SPAN_ID, 0x00000000DEADBEF0],
+    )
+    def test_generate_span_id_avoids_invalid(self, mock_getrandbits):
+        generator = RandomIdGenerator()
+        span_id = generator.generate_span_id()
+
+        self.assertNotEqual(span_id, span.INVALID_SPAN_ID)
+        self.assertGreater(span_id, span.INVALID_SPAN_ID)
```

Comment:
I think here you can use trace_api.INVALID_SPAN_ID and remove the import of span

Reply:
yes, your are right, thanks for this!